promql/functions_test.go

```diff
@@ -65,7 +65,7 @@ func TestDeriv(t *testing.T) {
 
 	vec, _ := result.Vector()
 	require.Len(t, vec, 1, "Expected 1 result, got %d", len(vec))
-	require.Equal(t, 0.0, vec[0].F, "Expected 0.0 as value, got %f", vec[0].F)
+	require.Zero(t, vec[0].F, "Expected 0.0 as value, got %f", vec[0].F)
```

Comment:
Zerof here

Maybe something to report to formatter with testifylint

Or maybe you didn't activate the sprintf setting to testifylint

Reply:
You didn't replyy question about the fact you used the formatter sprintf setting when using testifylint.

This one

https://github.com/Antonboom/testifylint?tab=readme-ov-file#3

My point was to know if there was a bug in testifylint or not